src/lib.rs

```diff
@@ -338,7 +338,7 @@ impl From<Host> for Target {
             Host::Aarch64UnknownLinuxMusl => Target::new_built_in("aarch64-unknown-linux-musl"),
             Host::Other(s) => Target::from(
                 s.as_str(),
-                &rustc::target_list(Verbosity::Quiet.into()).unwrap(),
+                &rustc::target_list(&mut Verbosity::Quiet.into()).unwrap(),
```

Comment:
Once again, we will never need to delete a line from the output here, so this isn't an issue.